ci/includes/pinned-tools.sh

```diff
@@ -25,11 +25,17 @@ function cleanUnity() {
 
 # Print the .NETCore version to aid debugging,
 # as well as ensuring that later calls to the tool don't print the welcome message on first run.
-dotnet --version
+if isMacOS; then
+  export DOTNET="/usr/local/share/dotnet/dotnet"
```

Comment:
Wouldn't it be easier to add this to path on macos when including this file?

Reply:
Keep in mind this won't just be running on our CI.
How about isolating this by adding `/usr/local/share/dotnet` to PATH iff `which dotnet` returns non-zero?

Reply:
The change in path would only persist during the run of the script doesnt it?

Reply:
Correct - my suggestion to avoid doing anything unless `dotnet` is completely unavailable on any existing PATH.

Reply:
Will make the change 👍